packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts

```diff
@@ -68,12 +51,16 @@ export const gridMiniMenuLinkButtonStyle = style([
     padding: '$5',
     textAlign: 'left',
     fontWeight: '$bold',
+    fontSize: '$sm',
   }),
   {
-    transition: 'all 0.1s ease',
     width: '100%',
-    borderBottom: `1px solid ${vars.colors.$borderDefault}`,
+    borderBottom: `1px solid #E3E1E5`,
```

Comment:
I guess these fixed colour values are missing from the design tokens?

Reply:
yes, I got them from Isa for the time being until they add all the new tokens in the ui library